homeassistant/components/dynalite/strings.json

```
@@ -5,8 +5,8 @@
         "description": "Please enter the connection information of your Dynalite gateway.",
         "data": {
           "name": "Name",
-          "host": "Gateway IP",
-          "port": "Port"
+          "host": "[%key:common::config_flow::data::host%]",
+          "port": "[%key:common::config_flow::data::port%]",
```

This is my bad, this shouldn't have a trailing comma.
```suggestion
          "port": "[%key:common::config_flow::data::port%]"
```

---

ok. i fixed it when it didnt run for me. pushing the changes now